plugins/polio/js/src/components/Inputs/ScopeInput.tsx

```diff
@@ -38,9 +39,13 @@ import sortBy from 'lodash/sortBy';
 
 import { FieldProps } from 'formik/dist/Field';
```

Comment:
I get a warning for this import  
![Screenshot 2022-11-15 at 09 36 58](https://user-images.githubusercontent.com/38907762/201870398-0a2c234e-5925-4cd4-92aa-d705870577ab.png)